pallets/automation-time/rpc/src/lib.rs

```diff
@@ -111,23 +113,27 @@ where
 		action: u8,
 		executions: u32,
 		at: Option<<Block as BlockT>::Hash>,
-	) -> Result<NumberOrHex> {
+	) -> RpcResult<NumberOrHex> {
 		let api = self.client.runtime_api();
 		let at = BlockId::hash(at.unwrap_or_else(||
 			// If the block hash is not supplied assume the best block.
 			self.client.info().best_hash));
 		let runtime_api_result =
-			api.get_time_automation_fees(&at, action, executions).map_err(|e| RpcError {
-				code: ErrorCode::ServerError(Error::RuntimeError.into()),
-				message: "Unable to get time automation fees".into(),
-				data: Some(format!("{:?}", e).into()),
+			api.get_time_automation_fees(&at, action, executions).map_err(|e| {
+				CallError::Custom(ErrorObject::owned(
+					Error::RuntimeError.into(),
+					"Unable to get time automation fees",
+					Some(e.to_string()),
+				))
 			})?;
 
 		let try_into_rpc_balance = |value: Balance| {
-			value.try_into().map_err(|_| RpcError {
-				code: ErrorCode::InvalidParams,
-				message: format!("RPC value doesn't fit in NumberOrHex representation"),
-				data: None,
+			value.try_into().map_err(|_| {
+				JsonRpseeError::Call(CallError::Custom(ErrorObject::owned(
+					Error::RuntimeError.into(),
+					"RPC value doesn't fit in NumberOrHex representation",
+					Some(format!("RPC value cannot be translated into NumberOrHex representation")),
```

Comment:
it doesn't like `None` because it can't serialize for some reason

Reply:
You can debug the error here if you want. Can be useful for printing stack trace (maybe more?)- `format!("{:?}", e)` where e is the arg to map_err

Reply:
chatted offline, error was `<Balance as TryInto<u64>>::Error` doesn’t implement `std::fmt::Debug`. would have to reimplement TryInto<u64>::Error, so just leaving this be for now